articles/portal/ptl-ref-release-notes.md

```diff
@@ -19,6 +19,16 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 1.144.0
+
+Date: 14th July 2020
+
+Feature update
```

Comment:
Is this more of a "Bug fix", not a feature update. I don't fully understand what is written in the details, so I not sure a customer will do either. Maybe we change this to something like "Fixed situation where a customer might be presented with a blank page when switching to a newly created account".

Reply:
Good point. I'll make these changes now.